Swift/Tests/PredictiveQueryTest+CoreML.swift

```diff
@@ -77,6 +77,7 @@ class PredictiveQueryWithCoreMLTest: CBLTestCase {
         try saveDocument(doc)
     }
 
+    #if COUCHBASE_ENTERPRISE
```

Comment:
Same as the previous comment. I think this file is only in the EE target so it shouldn't need COUCHBASE_ENTERPRISE.